### PR TITLE
[HIPFORGE-421] Upgrade to Experience Manager 15

### PR DESCRIPTION
**Pull request by nileshdeepak-bloomreach**

### CODE REVIEW

File: brut-resources/src/test/java/client/packagename/PageModelTest.java [hunk 1]
```diff
@@ -57,7 +57,6 @@ public void test() throws IOException {
         ObjectMapper mapper = new ObjectMapper();
         JsonNode jsonNode = mapper.readValue(response, JsonNode.class);
         assertTrue(jsonNode.get("page").size() > 0);
-        assertEquals(jsonNode.get("page").get("models").get("pageable").get("pageSize").intValue(), 10);
```
Comment (maxgda) [inline brut-resources/src/test/java/client/packagename/PageModelTest.java:60]: don't remove, adapt

File: pom.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.onehippo.cms7</groupId>
     <artifactId>hippo-cms7-project</artifactId>
-    <version>30.4.0</version>
+    <version>31.0.0</version>
```
Comment (maxgda) [inline pom.xml]: shouldn't this be 15.0.0?

File: brut-resources/src/test/java/client/packagename/PageModelTest.java (hunk 1 above)
Comment (nileshdeepak-bloomreach) [reply brut-resources/src/test/java/client/packagename/PageModelTest.java:60]: Updated.

File: pom.xml (hunk 2 above)
Comment (nileshdeepak-bloomreach) [reply pom.xml]: Updated.

**Comment by joeyliechty** [comment]
@nileshdeepak-bloomreach @maxgda tests pass successfully in the main dir of this repo.
_however_, the demo project no longer builds properly.
`cd demo && mvn clean verify`
is that expected?
I assume that this change should include a functional demo project, buildable just the same as it was in the master branch.
If it is expected to fail for some reason, there should be some notes in the various readme files explaining why it will not build and/or fixes to remedy the failure.

**Comment by joeyliechty** [comment]
@nileshdeepak-bloomreach i think i must have made a mistake with the move from j8 to j11 - as i am not able to reproduce the build issue in my local. apologies.

**Pull request merged**